packages/features/ee/workflows/pages/workflow.tsx

```diff
@@ -251,7 +251,7 @@ function WorkflowPage() {
           </div>
         }
         heading={
-          session.data?.hasValidLicense &&
+          session.data?.license.valid &&
```

Comment:
License validity within the session object has been moved to its own object together with the key in order to have it at hand for other usages.